File: tests/Generator/resource/swift/complex/TypeSchema.swift
// TypeSchema specification
class TypeSchema: Codable {
    var definitions: Dictionary<String, DefinitionType>
    var _import: Dictionary<String, String>
    var root: String

    enum CodingKeys: String, CodingKey {
        case definitions = "definitions"
        case _import = "import"
        case root = "root"
    }
}

File: tests/Generator/resource/csharp/complex/TypeSchema.cs
using System.Text.Json.Serialization;

/// <summary>
/// TypeSchema specification
/// </summary>
public class TypeSchema
{
    [JsonPropertyName("import")]
    public System.Collections.Generic.Dictionary<string, string>? Import { get; set; }

    [JsonPropertyName("definitions")]
    public System.Collections.Generic.Dictionary<string, DefinitionType>? Definitions { get; set; }

    [JsonPropertyName("root")]
    public string? Root { get; set; }

}

